systems/plants/dev/RigidBodySpatialForce.m

```diff
@@ -0,0 +1,53 @@
+classdef RigidBodySpatialForce < RigidBodyForceElement
+  
```

Comment:
This looks good to me.
Re:
```
+      % todo: compare this to RigidBodyThrust, which is comparitively much
+      % simpler.  Do I have a good way of computing the correct jacobian in
+      % one shot?
```
What you have is pretty much the fastest way, but a shorter version would be something like
```
[J, v_indices] = manip.geometricJacobian(kinsol, 0, obj.frame_id, obj.frame_id, false); % geometric Jacobian with base world, end effector body that frame is attached to, expressed in frame, in terms of v
B(v_indices, obj.input_num) = J';
```
(haven't tested or even run this code, but you get the idea)